quilt/data.py

```diff
@@ -62,6 +67,12 @@ def _data_keys(self):
         """
         return [name for name, child in self._items() if not isinstance(child, GroupNode)]
 
+    def data(self):
+        """
+        Returns the contents of the node: a dataframe or a file path.
+        """
+        return self._package.get_obj([getattr(self, name)._node for name in self._data_keys()])
```

Comment:
Having to call `getattr` seems silly. You could use `self._items()` and then check the types... Or maybe add a `_data_items()` method.